orderbook/src/fee.rs

```diff
@@ -23,6 +30,41 @@ pub struct MinFeeCalculator {
     unsupported_tokens: HashSet<H160>,
 }
 
+#[cfg_attr(test, mockall::automock)]
+#[async_trait::async_trait]
+pub trait MinFeeCalculating {
+    fn new(
```

Comment:
Does new have to be part of the interface? Usually the instantiation dependencies should be an implementation detail of each implementor and not dictated by the trait.